<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>heart</class>
 <widget class="QWidget" name="heart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/手动.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>89</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/加热.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>340</y>
     <width>89</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/风扇(2).png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/关雾化.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>-10</y>
     <width>1051</width>
     <height>621</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/背景图.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 italic 20pt &quot;Lohit Tamil&quot;;</string>
   </property>
   <property name="text">
    <string>人工控制</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 italic 20pt &quot;Lohit Tamil&quot;;</string>
   </property>
   <property name="text">
    <string>加热器</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>430</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 italic 20pt &quot;Lohit Tamil&quot;;</string>
   </property>
   <property name="text">
    <string>风扇</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 italic 20pt &quot;Lohit Tamil&quot;;</string>
   </property>
   <property name="text">
    <string>雾化器</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>420</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(28, 113, 216);
font: 75 italic 20pt &quot;Lohit Tamil&quot;;</string>
   </property>
   <property name="text">
    <string>start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopButton">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>420</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(28, 113, 216);
font: 75 italic 20pt &quot;Lohit Tamil&quot;;</string>
   </property>
   <property name="text">
    <string>stop</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>90</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::UpDownArrows</enum>
   </property>
   <property name="correctionMode">
    <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
   </property>
   <property name="calendarPopup">
    <bool>false</bool>
   </property>
   <property name="time">
    <time>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>190</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;Ubuntu&quot;;</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit3">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>290</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/1.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;Ubuntu&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>0</y>
     <width>331</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 63 42pt &quot;URW Bookman&quot;;
font: 63 66pt &quot;URW Bookman&quot;;
font: 95 italic 20pt &quot;Lohit Tamil&quot;;</string>
   </property>
   <property name="text">
    <string>定时喂食开灯</string>
   </property>
  </widget>
  <widget class="QLabel" name="tempEdit">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>520</y>
     <width>301</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Condensed</family>
     <pointsize>20</pointsize>
     <weight>9</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 italic 20pt &quot;Ubuntu Condensed&quot;;</string>
   </property>
   <property name="text">
    <string>已关闭定时喂食</string>
   </property>
  </widget>
  <widget class="QLabel" name="tempEdit_2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>520</y>
     <width>111</width>
     <height>34</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Condensed</family>
     <pointsize>20</pointsize>
     <weight>9</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 italic 20pt &quot;Ubuntu Condensed&quot;;</string>
   </property>
   <property name="text">
    <string>状态：</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>80</y>
     <width>81</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 italic 20pt &quot;Lohit Tamil&quot;;</string>
      </property>
      <property name="text">
       <string>早上</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 italic 20pt &quot;Lohit Tamil&quot;;</string>
      </property>
      <property name="text">
       <string>中午</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 75 italic 20pt &quot;Lohit Tamil&quot;;</string>
      </property>
      <property name="text">
       <string>晚上</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>listView</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_8</zorder>
  <zorder>pushButton_7</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>startButton</zorder>
  <zorder>stopButton</zorder>
  <zorder>timeEdit1</zorder>
  <zorder>timeEdit2</zorder>
  <zorder>timeEdit3</zorder>
  <zorder>label_9</zorder>
  <zorder>tempEdit</zorder>
  <zorder>tempEdit_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
